src/callAPIMethod.js

```diff
@@ -26,13 +26,16 @@ export default function callAPI(
 
     return fetch(url, {...defaults.fetchOptions, ...fetchOptions, ...options})
         .then( result => {
-            const promiseCallback = result.body ? result[method]() : new Promise(resolve => resolve(result.body));
+            const promiseCallback = result[method]();
```

Comment:
@ya-woodcutter according to https://developer.mozilla.org/en-US/docs/Web/API/Response
Response implements Body. I think the issue is with isomorphic-fetch not being really isomorphic and using node-fetch underneath. 
Did you try using Response.blob() instead?
```js
const promiseCallback =  result[result.body ? method : 'blob']();
```